### DISCUSSION TITLE
V2.75

### DISCUSSION THREAD

**Discussion by zhongyang219** [zhongyang219/MusicPlayer2#371] [Announcements]
* 新增FFMPEG播放内核（需要单独下载）
* 新增一些新的界面布局
* 界面3、界面4和界面5使用xml重制，优化界面4的效果
* 自定义界面中新增了堆叠组件
* 浮动播放列表的使用逻辑优化
* 修正写入ini时可能出现缓冲区不足的问题
* 在媒体库中新建播放列表时，新的播放列表显示到最前面
* 自绘控件跟随“使用圆角风格”的设置
* 修正“音效设定”中的“混响”无法保存的问题
* 新增自绘菜单栏
* 其他细节方面的改进

<hr /><em>This discussion was created from the release <a href='https://github.com/zhongyang219/MusicPlayer2/releases/tag/'>V2.75</a>.</em>

**Comment by jiffish** [comment]
播放列表，添加文件夹的时候，例如我的分类是 伍佰-某某年的专辑-歌曲，只能一个专辑一个专辑的添加，不能添加伍佰这个文件夹，自动读取十几个专辑。

**Reply by lrisora** [reply]
播放列表添加文件夹可以选择包含子文件夹，
这样就会递归添加所有子目录了

![圖片](https://user-images.githubusercontent.com/43085437/180630371-eeb5155f-53af-443d-bb42-c74dd993aa37.png)

**Comment by jiffish** [comment]
原来如此，谢谢，这我的确没注意到，哈哈。




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年7月24日(星期天) 中午11:04
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [zhongyang219/MusicPlayer2] V2.75 (Discussion #371)





 
播放列表添加文件夹可以选择包含子文件夹，
 这样就会递归添加所有子目录了

**Comment by staneliya** [comment]
歌词匹配比较蛋疼 都是一些垃圾歌词

**Comment by Mr-Liu1218** [comment]
大佬，什么时候能加入拖动歌词调节播放进度的功能啊

**Comment by YiGuangNB** [comment]
还是希望作者能加一个“按照CD号排序“的功能，音轨号用不上呀

**Comment by xiaojielv** [comment]
啥时候更新啊大佬

**Reply by zhoushengdao** [reply]
已经更新了
